dll/win32/shell32/stubs.cpp

```diff
@@ -643,15 +643,53 @@ SHChangeNotifySuspendResume(BOOL bSuspend,
     return FALSE;
 }
 
+// CreateEnvironmentBlock
+typedef BOOL (WINAPI *CREATEENVIRONMENTBLOCK)(LPVOID *, HANDLE, BOOL);
+
 /*
- * Unimplemented
+ * Implemented
  */
 EXTERN_C BOOL
 WINAPI
-RegenerateUserEnvironment(LPVOID *lpUnknown, BOOL bUnknown)
-{
-    FIXME("RegenerateUserEnvironment() stub\n");
-    return FALSE;
+RegenerateUserEnvironment(LPVOID *lpEnvironment, BOOL bUpdateSelf)
```

Comment:
It is no longer a stub, so it does not belong in `stubs.cpp` anymore :)

Reply:
Where is the better place?

Reply:
`shell32.cpp` or it's own file if there are going to be more environment related functions.

Reply:
Moved to shell32.cpp.